File: leetcode.com/0938_range_sum_of_bst/solution.py
class Solution:
    def rangeSumBST(self, root: Optional[TreeNode], low: int, high: int) -> int:
        total = 0
        q = collections.deque([root])

        while q:
            node = q.popleft()

            if not node:
                continue

            if low <= node.val <= high:
                total += node.val

            q.append(node.left)
            q.append(node.right)

        return total

File: leetcode.com/0938_range_sum_of_bst/solution.go
func rangeSumBST(root *TreeNode, low int, high int) int {
    if root == nil {
        return 0
    }

    sum := 0

    if low <= root.Val && root.Val <= high {
        sum += root.Val
    }

    return rangeSumBST(root.Left, low, high) + rangeSumBST(root.Right, low, high) + sum
}
